core_lib/src/interface/basedockwidget.cpp

```diff
@@ -17,10 +17,22 @@ GNU General Public License for more details.
 
 
 #include "basedockwidget.h"
+#include "platformhandler.h"
 
 BaseDockWidget::BaseDockWidget(QWidget* pParent) 
 : QDockWidget(pParent, Qt::Tool)
 {
+
+#ifdef __APPLE__
+    if (PlatformHandler::isDarkMode())
+    {
+        setStyleSheet("QDockWidget::title { background-image: url(:icons/darkmode-gradient.png); "
```

Comment:
Could we use QLinearGradient instead here? Ex: `background: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4e4f4d, stop: 1 #424241); }`

Reply:
done.